mlflow/models/container/__init__.py

```diff
@@ -91,8 +91,8 @@ def _install_pyfunc_deps(
     server_deps = ["gunicorn[gevent]"]
     if enable_mlserver:
         server_deps = [
-            "'mlserver>=1.2.0,!=1.3.1,<1.4.0'",
-            "'mlserver-mlflow>=1.2.0,!=1.3.1,<1.4.0'",
+            "'mlserver>=1.2.0,!=1.3.1'",
+            "'mlserver-mlflow>=1.2.0,!=1.3.1'",
```

Comment:
Could you add a constraint of "<2" ?  and also add this constraint into [pyproject.toml](https://github.com/mlflow/mlflow/blob/9b99927e01d599ecb9d2afadebf25dcf630ef9f8/pyproject.toml#L88) ?

This is to avoid the breaking changes introduced in mlserver>=2 in future.